EasyFinance.Server/EasyFinance.Server.csproj

```diff
@@ -10,6 +10,13 @@
     <SpaProxyServerUrl>https://localhost:4200</SpaProxyServerUrl>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="DTOs\**" />
+    <Content Remove="DTOs\**" />
+    <EmbeddedResource Remove="DTOs\**" />
+    <None Remove="DTOs\**" />
+  </ItemGroup>
+
```

Comment:
```suggestion
```

I suggest remove these lines because it can cause some errors during the build and publish, probably the VS added it automatically when you change the namespace.